tests/integration/backward_compatible/reconnect_test.py

```diff
@@ -244,6 +254,8 @@ def _verify_listeners_after_client_disconnected(self, member_address, client_add
             cluster_connect_timeout=sys.maxsize,
         )
 
+        self.client = client
+
```

Comment:
nit: could do this as well (same for other `self.client = client`):
```python
self.client = HazelcastClient(
            cluster_name=cluster.id,
            cluster_members=[client_address],
            cluster_connect_timeout=sys.maxsize,
 )
```

Reply:
Then I head to reference them as self.client instead of client, which I don't like